src/fillname/utils/parser.py

```diff
@@ -3,17 +3,21 @@
 """
 
 import logging
+import sys
 from argparse import ArgumentParser
 from textwrap import dedent
 from typing import Any, cast
 
-from pkg_resources import DistributionNotFound, require
-
 __all__ = ["get_parser"]
 
+if sys.version_info[1] < 8:
+    import importlib_metadata as metadata
+else:
+    from importlib import metadata
+
 try:
```

Comment:
Since the source layout kind of forces installation, we can probably avoid catching `metadata.PackageNotFoundError`.

Reply:
Agreed!